cilium/cmd/bpf_ct_list_test.go

```diff
@@ -0,0 +1,170 @@
+// Copyright 2020 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build !privileged_tests
+
+package cmd
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"os"
+	"testing"
+
+	"github.com/cilium/cilium/common/types"
+	"github.com/cilium/cilium/pkg/byteorder"
+	"github.com/cilium/cilium/pkg/command"
+	"github.com/cilium/cilium/pkg/maps/ctmap"
+	"github.com/cilium/cilium/pkg/tuple"
+	"github.com/cilium/cilium/pkg/u8proto"
+
+	. "gopkg.in/check.v1"
+)
+
+func Test(t *testing.T) { TestingT(t) }
+
+type BPFCtListSuite struct{}
+
+var _ = Suite(&BPFCtListSuite{})
+
+var (
```

Comment:
Aren't most of these constants?

Reply:
In theory yes, but the different initialisers used to set the values return non-const types, so I get e.g. (from `go vet`):

```
cilium/cmd/bpf_ct_list_test.go:43:13:
        (types.IPv4 literal) (value of type types.IPv4) is not constant
```

If I try to replace `var` with `const` here.